### ISSUE TITLE
(pytorch) Gen-LaneNet: a generalized and scalable approach for 3D lane detection

### ISSUE THREAD

**Issue by akshitagupta15june** [akshitagupta15june/lane_detection_opencv#45]
predict 3D lane from an image given known camera height and pitch angle. Pretrained models for the segmentation subnetwork and the 3D geometry subnetwork are loaded. Meanwhile, anchor normalization parameters wrt. the training set are also loaded.

**Comment by anushka17agarwal** [comment]
Hi, i would like to work on this issue please assign it to me.

**Comment by akshitagupta15june** [comment]
ok
